<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IAMSARScenarioForm</class>
 <widget class="QDialog" name="IAMSARScenarioForm">
  <property name="windowTitle">
   <string>Выбор сценария IAMSAR</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <widget class="QComboBox" name="scenarioSelector"/>
   </item>
   <item>
    <widget class="QTextEdit" name="scenarioDescription"/>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <widget class="QPushButton" name="buttonApply">
       <property name="text"><string>Применить</string></property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClose">
       <property name="text"><string>Отмена</string></property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>